airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -159,12 +159,84 @@ def get_start_timestamp(self, stream_state) -> int:
 
         return start_point
 
+class IncrementalStripeStreamWithUpdates(IncrementalStripeStream):
+    """
+        This is a base class for incremental streams that support updates using the events API
+        In first sync it will not get any updates as the records are already updated
+        After first sync it will get all updates starting from date of the last sync
+    """
+    event_types = None
+    update_field = "event_created"
+
+    def read_records(self,stream_slice, stream_state, **kwargs) -> Iterable[Mapping[str, Any]]:
+        # If this is not the first sync also get the updates
```

Comment:
minor: `# If this is not the first sync also get the updates` -> `# If this is not the first sync, capture updates from event stream for the resource`